src/handlers/processors.ts

```diff
@@ -35,7 +37,19 @@ export const processors: Record<string, Handler> = {
   },
   [GithubEvent.ISSUES_CLOSED]: {
     pre: [nullHandler],
-    action: [handleIssueClosed],
+    // Changed this because functions now returns string responses not void
+    action: [
+      async (): Promise<void> => {
+        const { payload: _payload } = getBotContext();
+        const issue = (_payload as Payload).issue;
+        try {
+          const comment = await handleIssueClosed();
+          return await addCommentToIssue(comment!, issue!.number);
+        } catch (err: any) {
+          return await addCommentToIssue("Error: " + err.message, issue!.number);
+        }
+      },
+    ],
```

Comment:
I don't agree to have it here. You can update action handler type properly or do whatever you can. but keep the previous one. 

`action: [handleIssueClosed]`

Reply:
Okay, i'd look for something